src/Organisms/EnhancedList/EnhancedList.stories.jsx

```diff
@@ -172,9 +172,9 @@ export const enhancedList = () => {
     { name: 'favoriteFruit', translationKey: 'Favorite fruit', description: 'Favorite fruit of the person. This tells a lot about one\'s state of mind' },
   ].map(c => ({ show: true, ...c })))
 
-  const setColumnShown = (columnName, willShow) => {
+  const setColumnShown = columnName => {
     setColumns(columns.map(({ name, show, ...column }) => (
-      { name, ...column, show: name === columnName ? willShow : show }
+      { name, ...column, show: name === columnName ? !show : show }
```

Comment:
No problem in a story but in real use we'd better do something along this to avoid regenerating all the columns each time one of them is enabled or disabled:
```js
const toggleColumn = name => setColumns(columns.map(column =>
  column.name === name ? { ...column, show: !column.show } :
  column
))
```